styles.css

```diff
@@ -0,0 +1,264 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  //display: flex;
+  //background-color: #ff8c00;

```

Comment:
Please remove unnecessary comments in your codebase. This makes code your unclean.